bin/varnishtest/vtc.c

```diff
@@ -746,7 +760,8 @@ cmd_feature(CMD_ARGS)
 				vtc_stop = 2;
 			}
 #endif
-		} else if (!strcmp(*av, "cmd")) {
+		}
+		else if (!strcmp(*av, "cmd")) {
```

Comment:
Although I personally prefer have `else` branches on the next line, as pointed out to me several times this is not the FreeBSD/Varnish code style.